lib/internal/modules/esm/loader.js

```diff
@@ -109,6 +110,28 @@ let hooksProxy;
  * @typedef {ArrayBuffer|TypedArray|string} ModuleSource
  */
 
+/**
+ * @typedef {Object} LoadResult
+ * @property {ModuleFormat} format
+ * @property {ModuleSource} source
+ */
+
+/**
+ * @typedef {Object} ResolveResult
+ * @property {string} format
+ * @property {URL['href']} url
+ */
+
+/**
+ * @typedef {Object} Customizations
```

Comment:
I think you want `object` (little `o`)

Reply:
I think this definition already exists in customization_hooks too

Reply:
I fixed the first one, couldn't find a definition in `customization_hooks`